users/models/favorite.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.auth.models import User
+from articles.models import Article
+
+class Favorite(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    article_id = models.ForeignKey(Article, on_delete=models.CASCADE)
```

Comment:
Lo dejaria sólo en article, no article_id, por simplicidad.

Reply:
clave foránea